src/db/migrations/migration1.js

```diff
@@ -0,0 +1,25 @@
+const logger = require('../../utils/logger');
+const DBHelper = require('../db-helper');
+const { TX_STATUS } = require('../../constants');
+
+async function migration1() {
+  try {
+    const events = await DBHelper.findEvent({ txStatus: TX_STATUS.WITHDRAW });
```

Comment:
cant we just loop through all the `successful` withdraws instead of having to fetch events first? you will have to alter the `updateEvent` func to be able to look it up by `address` as well.

Reply:
so update a withdrawer one at a time, instead of updating an event withdrawnList as a whole in one time?

Reply:
If I understand that correctly, why? I thought the fewer update the quicker?

Reply:
Oops i forgot to submit the update. How about this solution?
1. Fetch all successful withdraws
2. Create a map: eventAddress -> [] of withdraw address
3. Populate map as you iterate withdraw list
4. Iterate map keys and update events so they have the full withdrawn list
This should lower the number of calls to: `findWithdraws + (updateEvent * numEvents)` vs `findEvent + (findWithdraws * numEvents) + (update * numEvents)`